server/src/main/resources/resources/attestation/private_key.pem

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Do we have a plan to move this out of the open source repository?

Reply:
No, this is just an arbitrary key, it does not give you any powers. I added a comment. We actually do have a key which is registered with Funke (although it's moot as they do not check client attestation yet). But this key is not Funke-registered key.